src/gui/book/Book.cpp

```diff
@@ -51,7 +51,7 @@ class PlayerBookPage {
 public:
 	void playReleaseSound();
 	void playErrorSound();
-	void ARX_INTERFACE_ManageOpenedBook_LeftTabs(bool tabVisibility[10], long & activeTab);
+	void drawLeftTabsCommon(bool tabVisibility[10], long & activeTab);
```

Comment:
This is also updates state besides drawing so the function name is a bit misleading - maybe it should be split?